### DISCUSSION TITLE
【求助】ui界面不对劲

### DISCUSSION THREAD

**Discussion by FeiXiao-zm** [share121/inter-knot#775] [其他]
本来用的好好的，不知道怎么变大了  还是开了电脑模式  原版直接看不了
![Image_1723278709089](https://github.com/user-attachments/assets/2c097f01-d641-4434-84f3-40ccfb97fe3c)
![Screenshot_20240810_162202_mark via](https://github.com/user-attachments/assets/95969f6e-4eb6-4c9f-bace-cb273fb5ed17)
![Screenshot_20240810_162156_mark via](https://github.com/user-attachments/assets/7edb2330-d4d8-4ac9-b543-4cdcfe108663)

**Comment by FeiXiao-zm** [comment]
可恶啊，各位有没有什么解决办法

**Comment by Callkit-P0p** [comment]
或许可以下载Steam++  加速一下试试，
再不济 就清除浏览器缓存重新进入

**Reply by aiTunes-dev** [reply]
看样子楼主用的是手机的via 手机好像没有Steam++

**Reply by Callkit-P0p** [reply]
有啊，Steam++有Android版

**Reply by FeiXiao-zm** [reply]
是的。。是手机版  唔…大佬有渠道吗 不太懂这些😔

**Comment by keerdun** [comment]
你UI怎么那么大的，我的好小只能一行容纳两个帖子

**Reply by FeiXiao-zm** [reply]
唔…不知道

**Comment by GanlinSama** [comment]
缺加载器的话我这边推荐Clash for windows，也有支持安卓端的版本

**Reply by FeiXiao-zm** [reply]
好的！谢谢老师

**Comment by HCYYB** [comment]
这个委托我接了！
浏览器搜索“瓦特工具箱”（steam++或者watt toolkit）
根据设备下载
加速“Gihub”
进入绳网并刷新

**Reply by FeiXiao-zm** [reply]
谢谢大佬！！！！！

**Comment by SDCOM-0415** [comment]
这个委托我接了！
我推荐直接用梯子
[地址一](http://piankelaw.top "地址一")
[地址二](http://zheke.top "地址二")
这是**我自己卖**的
#807

**Reply by FeiXiao-zm** [reply]
（震惊）（捂嘴）（滑跪）绳匠，你是我的神！！！

**Comment by aaa00073** [comment]
用steam++试试

**Reply by FeiXiao-zm** [reply]
嗯…在下载了  好慢   这个能解决ui问题吗

**Comment by share121** [comment]
修复方法，见 https://github.com/share121/inter-knot/discussions/1100

**Reply by FeiXiao-zm** [reply]
好的好的，辛苦了

**Reply by FeiXiao-zm** [reply]
可恶啊  回不到一排五个的了

**Comment by FeiXiao-zm** [comment]
可恶  回不到最开始的样子了
